src/rover/camera.cpp

```diff
@@ -0,0 +1,229 @@
+#include <fcntl.h>
```

Comment:
I like it when includes are alphabetized. http://alphabetizer.flap.tv/ is good at that